.github/workflows/release-minor.yml

```diff
@@ -18,13 +17,31 @@ jobs:
       - name: Push to master
         uses: ad-m/github-push-action@master
         with:
-          github_token: ${{ secrets.GH_RW_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Using `GITHUB_TOKEN` instead of `GH_RW_TOKEN` allow us to control other workflows listening for push to branch such as `release` and `forward-merge`. We don't want them running after `prerelease/minor` branch has been merged to `master`.